packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1486,14 +1502,28 @@ export class WindowsBuildImage implements IBuildImage {
     imagePullPrincipalType: ImagePullPrincipalType.CODEBUILD,
   });
 
+  /**
+   * The standard CodeBuild image `aws/codebuild/windows-base:2019-1.0`, which is
+   * based off Windows Server Core 2019.
+   */
+  public static readonly WIN_SERVER_CORE_2019_BASE: IBuildImage = new WindowsBuildImage({
+    imageId: 'aws/codebuild/windows-base:2019-1.0',
+    imagePullPrincipalType: ImagePullPrincipalType.CODEBUILD,
+    imageType: WindowsImageType.SERVER_2019,
+  });
+
   /**
    * @returns a Windows build image from a Docker Hub image.
    */
-  public static fromDockerRegistry(name: string, options: DockerImageOptions): IBuildImage {
+  public static fromDockerRegistry(
+    name: string,
+    options: DockerImageOptions = {},
+    imageType: WindowsImageType = WindowsImageType.STANDARD): IBuildImage {
```

Comment:
Add an empty line here, since the function body is equally indented with its signature.